packages/components/pie-cookie-banner/src/defs.ts

```diff
@@ -148,7 +180,22 @@ export type DefaultProps = ComponentDefaultProps<CookieBannerProps>;
 export const defaultProps: DefaultProps = {
     hasPrimaryActionsOnly: false,
     defaultPreferences: {},
-    locale: defaultLocale,
+    country: Country.GREAT_BRITAIN,
+    language: Language.ENGLISH,
     cookieStatementLink: '',
     cookieTechnologiesLink: '',
 };
+
+export const defaultLanguage = new Map<string, string>([
```

Comment:
I don't think it's required for this PR but it would be good if we could strengthen up the types here instead of string, we could have some union types for CountryCode, LanguageCode, etc.

`type LanguageCode = 'bg' | 'ca' | 'da' ...` etc